api/src/main/java/com/epam/pipeline/manager/scheduling/RunScheduler.java

```diff
@@ -68,42 +78,50 @@ public void init() {
+                    + schedule.getType() + ID + schedule.getSchedulableId());
             final JobDetail jobDetail = jobDetail(schedule);
 
             log.debug("Creating trigger for key " + jobDetail.getKey().getName() + " at date: " + DateUtils.now());
             final Trigger cronTrigger = createCronTrigger(schedule);
 
             final Date scheduledDate = quartzScheduler.scheduleJob(jobDetail, cronTrigger);
-            log.debug("Job for run: " + schedule.getRunId() + " scheduled successfully for date: " + scheduledDate);
+            log.debug("Job for: " + schedule.getType() + ID
+                    + schedule.getSchedulableId() + " scheduled successfully for date: " + scheduledDate);
         } catch (SchedulerException | ParseException e) {
-            log.error("SchedulerException while scheduling job for run " + schedule.getRunId() + " : " +
+            log.error("SchedulerException while scheduling job for run " + schedule.getSchedulableId() + " : " +
                       e.getMessage());
         }
     }
 
     public void unscheduleRunSchedule(final RunSchedule schedule) {
         try {
-            log.debug("Request received to unscheduling trigger for run " + schedule.getRunId());
+            log.debug("Request received to unscheduling trigger for: " + schedule.getType()
+                    + ID + schedule.getSchedulableId());
 
             final JobKey key = jobDetail(schedule).getKey();
 
             quartzScheduler.deleteJob(key);
 
-            log.debug("Schedule " + schedule.getCronExpression() + " for run " + schedule.getRunId() +
-                      " was revoked successfully.");
+            log.debug("Schedule " + schedule.getCronExpression() + " for "  + schedule.getType()
+                    + ID + schedule.getSchedulableId() + " was revoked successfully.");
         } catch (SchedulerException e) {
-            log.error("SchedulerException while unscheduling trigger for run " + schedule.getRunId() + " : " +
-                      e.getMessage());
+            log.error("SchedulerException while unscheduling trigger for "  + schedule.getType()
+                    + ID + schedule.getSchedulableId() + " : " + e.getMessage());
         }
     }
 
     private JobDetail jobDetail(final RunSchedule runSchedule) {
         JobDetailFactoryBean jobDetailFactory = new JobDetailFactoryBean();
-        jobDetailFactory.setJobClass(RunScheduleJob.class);
-        jobDetailFactory.getJobDataMap().put("RunId", runSchedule.getRunId());
+        jobDetailFactory.setJobClass(runSchedule.getType() == ScheduleType.PIPELINE_RUN
+                ? RunScheduleJob.class
+                : ConfigurationScheduleJob.class);
+        jobDetailFactory.getJobDataMap().put("SchedulableId", runSchedule.getSchedulableId());
+        jobDetailFactory.getJobDataMap().put(
+                "UserToken", authManager.issueToken(userManager.loadUserContext(runSchedule.getUser()), null).getToken()
```

Comment:
Method `authManager.issueToken` with `null` `expiration` will issue a token with default expiration (1 month by default). We need to specify expiration manually, if we need a long-term token